common/djangoapps/student/models.py

```diff
@@ -222,60 +222,19 @@ def user_by_anonymous_id(uid):
         return None
 
 
-def is_username_retired(username):
```

Comment:
I'm just moving this out into the `models_api.py` file in accordance with [this](https://github.com/edx/edx-platform/blob/master/docs/decisions/0002-inter-app-apis.rst) ADR.